java/vector/src/main/java/org/apache/arrow/vector/dictionary/Dictionary.java

```diff
@@ -52,14 +52,6 @@ public String toString() {
   }
 
   @Override
-  public boolean equals(Object o) {
```

Comment:
@elahrvivaz I removed this because when doing integration tests, the FieldVector `dictionary` wasn't being properly compared.  I think to compare Dictionaries, it's better to just separately compare the DictionaryEncoding and FieldVector, which is what I'm doing now in Integration.java.  Does that sound right to you?